WooCommerce/Classes/ViewRelated/Products/Edit Product/Downloads/Downloadable FIle List/ProductDownloadDnD.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+import CoreServices
+import Yosemite
+
+/// A wrapper around ProductDownload, to make it compatible in using as Drag and Drop data source in an table view. Represents a ProductDownload entity.
+///
+public class ProductDownloadDnD: NSObject, Codable {
+    public let download: ProductDownload
+
+    /// ProductDownloadDnD initializer.
+    ///
+    public init(download: ProductDownload) {
+        self.download = download
+    }
+
+    /// Public initializer for ProductDownloadDnD
+    ///
+    required public convenience init(from decoder: Decoder) throws {
```

Comment:
I don't think that we need `required public` 🤔 
```suggestion
    convenience init(from decoder: Decoder) throws {
```

Reply:
Yes, according to the above changes, we don't need it